airbyte-integrations/connectors/source-pipedrive/source_pipedrive/manifest.yaml

```diff
@@ -0,0 +1,348 @@
+version: "0.29.0"
+
+definitions:
+  selector:
+    type: RecordSelector
+    extractor:
+      type: DpathExtractor
+      field_path: ["data"]
+
+  selector_increment:
+    type: RecordSelector
+    extractor:
+      type: DpathExtractor
+      field_path: ["data", "*", "data"]
+      
+  selector_users:
+    type: RecordSelector
+    extractor:
+      type: DpathExtractor
+      field_path: ["data", "*", "data","*"]
+
+  requester:
+    type: HttpRequester
+    url_base: "https://api.pipedrive.com/v1"
+    http_method: "GET"
+    request_parameters:
+      api_token: "{{config['api_token']}}"
+      limit: "50"
+      items: "{{ parameters.path_extractor }}"
+  retriever:
+    type: SimpleRetriever
+    record_selector:
+      $ref: "#/definitions/selector"
+    paginator:
+      type: "DefaultPaginator"
+      pagination_strategy:
+        type: "CursorPagination"
+        cursor_value: "{{ response['additional_data']['pagination']['next_start'] }}"
+      page_token_option:
+        type: "RequestOption"
+        inject_into: "request_parameter"
+        field_name: "start"
+    requester:
+      $ref: "#/definitions/requester"
+
+  retriever_increment:
+    $ref: "#/definitions/retriever"
+    record_selector:
+      $ref: "#/definitions/selector_increment"
+
+  incremental_base:
+    type: DatetimeBasedCursor
+    cursor_field: "update_time"
+    datetime_format: "%Y-%m-%d %H:%M:%S"
+    start_datetime:
+      datetime: "{{ config['replication_start_date'] }}"
```

Comment:
You need to format this to `%Y-%m-%d %H:%M:%S`. The `replication_start_date` today is expected to be `%Y-%m-%dT%H:%M:%S`.